content/src/content/jcr_root/apps/core/wcm/sandbox/commons/ui/clientlibs/panelcontainer/js/PanelContainer.js

```diff
@@ -0,0 +1,169 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2018 Adobe Systems Incorporated
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+/* global CQ */
+(function($, ns, channel, window, undefined) {
+    "use strict";
+
+    var GET_DATA_SUFFIX = ".model.json";
+    var POST_SUFFIX = ".children.html";
+    var MESSAGE_ID = "cmp.panelcontainer";
+
+    /**
+     * @typedef {Object} PanelContainerConfig Represents a Panel Container configuration object
+     * @property {String} path The path to the Panel Container component represented
+     * @property {PanelContainerType} [panelContainerType] The Panel Container Type definition
+     * @property {HTMLElement} [el] The HTMLElement in the Granite.author.ContentFrame related to this panel container
+     */
```

Comment:
To which code does these docs relate? Should it be removed?

Reply:
It's the type definition for a configuration to a panel container, they should stay